### PR TITLE
Getting queues by priority range

### PR DESCRIPTION
**Pull request by SkrebtsovHeorhii**

### CODE REVIEW

File: src/Jobs/Reservers/PriorityReserver.php [hunk 1]
```diff
@@ -25,6 +31,16 @@ class PriorityReserver extends AbstractReserver implements ReserverInterface
...
 
+    /**
+     * Default min processed priority
+     */
+    const DEFAULT_MIN_PRIORITY = 1;
+
+    /**
+     * Default max processed priority
+     */
+    const DEFAULT_MAX_PRIORITY = 8;
```
Comment (osadchyi-s) [inline src/Jobs/Reservers/PriorityReserver.php]: Why 8? Better 10.

**Comment by osadchyi-s** [comment]
Please, add tests for different new cases: set only min priority, set only max priority, set minimum and maximum priority etc.

File: src/Jobs/Reservers/PriorityReserver.php (hunk 1 above)
Comment (SkrebtsovHeorhii) [reply src/Jobs/Reservers/PriorityReserver.php]: fixed

**Comment by SkrebtsovHeorhii** [comment]
> Please, add tests for different new cases: set only min priority, set only max priority, set minimum and maximum priority etc.

Added

**Review by osadchyi-s — APPROVED** [review]

**Review by sergeyklay — APPROVED** [review]
